src/components/dashboard.jsx

```diff
@@ -12,12 +12,24 @@ import CurrentUserStore from '../user-store';
 import AsyncButton from './async-button';
 import Time from './time';
 
-const SAMPLE_REPOS = [
+let SAMPLE_REPOS = [
   {repoOwner: 'huboard', repoName: 'huboard'},
   {repoOwner: 'openstax', repoNames: ['tutor-js', 'tutor-server'], comment: ' (multiple repositories)'},
   {repoOwner: 'jquery', repoName: 'jquery'}
 ];
 
+try {
+  const repos = REPOSITORIES;
+  if (repos) {
```

Comment:
Use double negation characters `!!repos` to convert it to a boolean. This is much safer.

Reply:
Hmm, It makes code harder to read and understand, and it is not used anywhere in the original project.

Reply:
Well, if you are fine with it then ok.